Makefile

```diff
@@ -1,5 +1,3 @@
-SHELL=/bin/bash -O extglob -c
```

Comment:
Restructuring the project directory renders this unnecessary and now this can be built on any shell.